airbyte-integrations/connectors/destination-motherduck/destination_motherduck/destination.py

```diff
@@ -241,8 +244,20 @@ def check(self, logger: logging.Logger, config: Mapping[str, Any]) -> AirbyteCon
             if CONFIG_MOTHERDUCK_API_KEY in config:
                 duckdb_config["motherduck_token"] = str(config[CONFIG_MOTHERDUCK_API_KEY])
                 duckdb_config["custom_user_agent"] = "airbyte"
-
-            con = duckdb.connect(database=path, read_only=False, config=duckdb_config)
+                # Next, we want to specify 'saas_mode' for during check,
+                # to reduce memory usage from unnecessary extensions
+                if "?" in path:
+                    # There are already some query params; append to them.
+                    path += "&saas_mode=true"
+                else:
+                    # No query params yet; add one.
+                    path += "?saas_mode=true"
+
+            con = duckdb.connect(
+                database="{path}&",
```

Comment:
```suggestion
                database=path,
```

Reply:
Was this intended to be an f-string?

Reply:
Good catch! 🙏 

I started changing it here, them moved it above when I realized I needed an if/then.